api/HTMLAreaElement.json

```diff
@@ -589,10 +589,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": false
```

Comment:
This `type` entry should also be removed, no such thing: https://html.spec.whatwg.org/#htmlareaelement

Reply:
Opened as #7540.